package.json

```diff
@@ -56,7 +63,8 @@
   "scripts": {
     "test": "cross-env NODE_ENV=test mocha --recursive test",
     "eslint": "eslint .",
-    "cover": "cross-env NODE_ENV=cover istanbul cover _mocha --recursive test"
+    "cover": "cross-env NODE_ENV=cover istanbul cover _mocha --recursive test",
+    "karma": "karma start"
```

Comment:
- change this to `test-client`
- add another for `test-server`
- make `npm test` run both the server and client tests